src/feedback.js

```diff
@@ -259,10 +261,18 @@ var feedbackFactory = function feedbackFactory($container, config) {
                 ? this.config.timeout[this.config.level]
                 : this.config.timeout;
 
-            $closer.off('click').on('click', function(e) {
-                e.preventDefault();
-                self.destroy();
-            });
+            $closer
+                .off('click')
+                .on('click', function(e) {
+                    e.preventDefault();
+                    self.destroy();
+                })
+                .off('keyup')
+                .on('keyup', function(e) {
+                    if ([codeEnter, codeSpace].includes(e.which)) {
```

Comment:
Caution, `includes` is not supported by IE11. Please check it is properly polyfilled by the bundling process. Otherwise, you should use something else.

Aso, `event.which` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which) and should not be used anymore. Prefer `event.key` instead.

Reply:
From [jQuery documentation](https://api.jquery.com/keyup/)
> To determine which key was pressed, examine the event object that is passed to the handler function. While browsers use differing properties to store this information, jQuery normalizes the .which property so you can reliably use it to retrieve the key code. This code corresponds to a key on the keyboard, including codes for special keys such as arrows.

Hope, it'd be fixed there. Anyway it'a good point to use `event.key` and `event.code`. But it doesn't use ascii codes for enter and space keys, so I don't wan't to focus on it so far